### PR TITLE
v0.1.0 release

### PR DESCRIPTION
**Pull request by mitokic**
final updates before creating v0.1.0 and submitting to CRAN

- add default behavior to run_ensemble_models to turn them off for daily/weekly forecasts

 - add best model column to hierarchical back tests

 - If any function arguments have an automated component, where the value changes based on other arguments, then the default value should be NULL instead of values like "auto". This complies with other R packages.

 - remove r check ci/cd for Win OS R3.6

 - Add google analytics tracker to packagedown site to measure traffic
 
 - remove any notes generated by R CMD Check, outside of 1 note related to how many packages are imported

 - Update package version

 - Add NEWS section

### CODE REVIEW

**Comment by mitokic** [comment]
Ah the classic "it works on my machine"....seems like an update to modeltime or timetk. Will investigate and fix before an official review. So close to CRAN submission!

**Comment by mitokic** [comment]
problem was with an update to the timetk::pad_by_time function. The update makes it run faster, but removes the previous functionality of being able to pad missing values with 0 within dplyr groups. Will be submitting a bug to the timetk github. Currently have a workaround, and will then update in Finn once the bug is fixed in timetk. New commit inbound

**Comment by mitokic** [comment]
ready to final review @AKannanMSFT

**Comment by mitokic** [comment]
ok final final commit before merge and CRAN submission :)

File: R/forecast_time_series.R [hunk 1]
```diff
@@ -315,7 +324,7 @@ forecast_time_series <- function(input_data,
   }
   
   # parallel run within azure batch
-  if(parallel_processing=="azure_batch") {
+  if(sum(parallel_processing=="azure_batch") == 1) {
```
Comment (AKannanMSFT) [inline R/forecast_time_series.R]: Thank you @Mike, Please fix this as discussed.

**Review by AKannanMSFT — COMMENTED** [review]
Changes to be made to improve readibility.

File: R/forecast_time_series.R (hunk 1 above)
Comment (mitokic) [reply R/forecast_time_series.R]: updated PR based on comments

**Review by AKannanMSFT — APPROVED** [review]
Congratulations on getting to v0.1

**Pull request merged**